PrototypeArmoury/Src/PrototypeArmoury/Classes/PA_DataStructures.uc

```diff
@@ -32,5 +34,11 @@ struct AutoItemConversion
 	var int Supplies;
 	var int Alloys;
 	var int Elerium;
+	var string DLC;
 };
 
+struct RequiredDLC
+{
+	var name ItemName;
+	var string DLC;
+};
```

Comment:
The name of this struct is super unrepresentative of what it's used for. Change to something like `ItemFromDLC`/`ItemWithRequiredDLC`/etc